packages/zero-next-plugin/package.json

```diff
@@ -1,13 +1,29 @@
 {
   "name": "@mui/zero-next-plugin",
-  "version": "0.0.1",
+  "version": "0.0.1-alpha.0",
```

Comment:
I don't think we need the `alpha` if we start with `0.0.x`

https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase